uac-client/src/main/java/com/bazaarvoice/emodb/uac/client/UserAccessControlClient.java

```diff
@@ -30,12 +30,14 @@
 import com.google.common.base.Objects;
 import com.google.common.base.Strings;
 
+import java.io.InputStream;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.io.InputStream'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.qa.bazaarvoice.com/coding_rules#rule_key=squid%3AUselessImportCheck)